backend/src/main/java/capstone/backend/model/db/order/OrderItem.java

```diff
@@ -18,11 +18,17 @@
 @Entity
 public class OrderItem {
 
+    public OrderItem(Product product,int quantity ){
```

Comment:
Constructors should be placed between attributes and methods. See https://www.oracle.com/technetwork/java/codeconventions-150003.pdf